packages/message-manager/src/AbstractMessageManager.ts

```diff
@@ -283,22 +283,31 @@ export abstract class AbstractMessageManager<
   }
 
   /**
-   * Sets the message to a new status via a call to this.setMsgStatus and
-   * updates the rawSig field in this.messages.
+   * Sets the message via a call to this.setResult and updates status of the message.
    *
    * @param messageId - The id of the Message to sign.
    * @param rawSig - The data to update rawSig in the message.
    * @param status - The new message status.
    */
   setMessageStatusAndResult(messageId: string, rawSig: string, status: string) {
+    this.setResult(messageId, rawSig);
```

Comment:
A minor one, but probably best to not use the method here as then we'd get two calls to `updateMessage` so duplicate logic and two `updateBadge` events.